plugins/generic/usageEvent/PKPUsageEventPlugin.inc.php

```diff
@@ -361,12 +340,11 @@ protected function getUsageEventData($hookName, $hookArgs, $request, $router, $t
 					$assocType = Application::getContextAssocType();
 					$canonicalUrlOp = '';
 					$canonicalUrlPage = 'index';
-					$downloadSuccess = true;
```

Comment:
could it be that this loop (i`f (($page == 'index' || empty($page)) && $op == 'index')`) is possible also if the pubObject is not a Context? Will the event then be recorded?